java/README.md

```diff
@@ -64,26 +64,46 @@ and arrow-format into a single JAR.  Using the classifier "shade-format-flatbuff
 pom.xml will make use of this JAR, you can then exclude/resolve the original dependency to
 a version of your choosing.
 
+### Updating the flatbuffers generated code
+
+To update the checked-in flatbuffer generated code perform the following:
+
+```bash
+cd $ARROW_HOME
+
+# remove the existing files
+rm -rf java/format/src
+
+# regenerate from the .fbs files
+flatc --java -o java/format/src/main/java format/*.fbs
```

Comment:
it is probably good to document version here?

Reply:
updated the readme to include a step to verify `flatc` matches the declared dependency version